pkg/plugins/plugindef/plugindef.cue

```diff
@@ -233,33 +222,64 @@ seqs: [
 					...
 				}
 
-				// For data source plugins, if the plugin supports logs.
+				// For data source plugins, if the plugin supports logs. Not used in Grafana core but may be used by external plugins.
```

Comment:
A bit confused on this one - should we still support this?

Reply:
I didn't remove it since there are still references to this in the code. The [DataSourcePicker](https://github.com/grafana/grafana/blob/main/packages/grafana-runtime/src/components/DataSourcePicker.tsx#L147) allows to receive this property `logs` and will only show data sources that enable this but I have not found any where in which a `DataSourcePicker` is instantiated with this property set to true (but it could be the case for some plugin in the wild).

Do you think we should remove it?

Reply:
Ah I get it now! I guess we keep it until we have a better handle on its usage. I think we can probably remove the _"Not used in Grafana core but may be used by external plugins."_ as this'll go stale as soon as someone decides to use it and someone else might go ahead and remove it 😨

Reply:
If we don't add anything, it's not clear at all what happens if people use it or not :disappointed: That's what I wanted to change in these PR. What if we say "It may be used to filter logs-only features"? That reflects that it's not clearly used atm and its design purpose.

Reply:
That sounds good! Some way to summarize or at least make consistent how we describe `metrics`, `logs`, `tracing`, etc. would be great, but I think this is enough for now!

Reply:
The problem is that the usage is not consistent :sweat_smile: For example, `metrics` is basically mandatory, `logs` is not used, `streaming` is used to render an aditional button in Explore... There is no limitation or requirement here.